source/user-manual/capabilities/malware-detection/virus-total-integration.rst

```diff
@@ -325,16 +329,16 @@ While the two previous alerts represent errors that might occur, the following a
             "found":"1",
             "malicious":"0",
             "source":{
-               "alert_id":"1668705721.82254",
+               "alert_id":"1712235120.697470",
                "file":"/media/user/software/suspicious-file10.exe",
                "md5":"d41d8cd98f00b204e9800998ecf8427e",
                "sha1":"da39a3ee5e6b4b0d3255bfef95601890afd80709"
             },
             "sha1":"da39a3ee5e6b4b0d3255bfef95601890afd80709",
-            "scan_date":"2022-11-17 17:19:48",
+            "scan_date":"2024-04-05 13:03:09",
             "positives":"0",
             "total":"60",
-            "permalink":"https://www.virustotal.com/gui/file/e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855/detection/f-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855-1668705588"
+            "permalink":""https://www.virustotal.com/gui/file/d41d8cd98f00b204e9800998ecf8427e/detection""
```

Comment:
Is this correct? (extra double quotes)

![image](https://github.com/wazuh/wazuh-documentation/assets/14913942/aa741776-73d9-41ab-bf37-19b1acacd00a)

Also, the JSON output has a different style (probably due to the double quotes)

![image](https://github.com/wazuh/wazuh-documentation/assets/14913942/7663991f-b2a8-45f8-9e68-edfa3e104ffa)

Reply:
It was corrected by leaving only one double quote. cc: @rauldpm